cleanWeather-vue/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+	"title": "CleanWeather",
```

Comment:
Maybe something a bit more descriptive here, like “Build a Weather App”.

Reply:
For example Groceries is called “Build a Full App”.

Reply:
oh I thought that was the actual name of the app...